geotrek/core/tests/test_commands.py

```diff
@@ -229,3 +230,353 @@ def test_load_paths_within_spatial_extent_no_srid_geom(self):
+        self.path_1_b.geom = LineString(Point(700050, 6600050), Point(700100, 6600100), Point(700050, 6600100),
+                                        Point(700000, 6600050), srid=settings.SRID)
+        self.path_1_b.save()
+        self.assertEqual(list(PathAggregation.objects.filter(topo_object=topo).values_list('order', flat=True)),
+                         [0, 1, 1, 2, 2, 2, 3, 3, 4, 4])  # /!\ Duplicated Point
+        call_command('reorder_topologies')
+        geometries = self.get_geometries()
+        self.assertEqual(geometries, [LineString((700000, 6600000), (700050, 6600050), srid=2154),
+                                      LineString((700050, 6600050), (700025, 6600075), srid=2154),
+                                      Point(700025, 6600075, srid=2154),
+                                      LineString((700025, 6600075), (700050, 6600050), srid=2154),
+                                      LineString((700050, 6600050), (700100, 6600100),
+                                                 (700050, 6600100), (700025, 6600075), srid=2154),
+                                      Point(700025, 6600075, srid=2154),
+                                      LineString((700025, 6600075), (700000, 6600050), srid=2154)
+                                      ]
+                         )
+
+        self.assertEqual(list(PathAggregation.objects.filter(topo_object=topo).values_list('order', flat=True)),
+                         [0, 1, 2, 3, 4, 5, 6])
+
+    def test_split_reorder_fail(self):
+        """
+        Part A
+
+        ⠳                   🡥
+          ⠳               🡥
+            ⠳           🡥
+        ⠳     0       🡥
+          ⠳     🡤   🡥
+            ⠳     X              🡥  Topo 1
+              ⠳ 🡥   ⠳            0  Topo 1 (point)
+              🡥 ⠳     ⠳          x  Topo 1 (2 directions)
+            🡥     ⠳     ⠳        ⠳  Paths (1 2)
+        1 🡥         ⠳ 3   ⠳ 2
+        🡥             ⠳     ⠳
+
+        Part B
+
+        FAILED
+
+        """
+        topo = TrekFactory.create(paths=[(self.path_1_a, 0, 1),
+                                         (self.path_2_a, 1, 0.5),
+                                         (self.path_2_a, 0.5, 0.5),
+                                         # (self.path_2_a, 0.5, 1), Doesn't exist in this test => MultiLinestring
+                                         (self.path_1_b, 0, 1)])
+        PathFactory.create(geom=LineString(Point(700000, 6600090), Point(700090, 6600000), srid=settings.SRID))
+        self.assertEqual(list(PathAggregation.objects.filter(topo_object=topo).values_list('order', flat=True)),
+                         [0, 0, 1, 2, 3])
+        output = StringIO()
+        call_command('reorder_topologies', stdout=output)
+        self.assertEqual(f'Topologies with errors :\nTREK id: {topo.pk}\n', output.getvalue())
+        geometries = self.get_geometries()
+        self.assertEqual(geometries, [LineString((700045, 6600045), (700050, 6600050), srid=2154),
```

Comment:
You have added assertions that test bad values ;) IMHO I would stop the test after asserting the error.